reana_cluster/cli/__init__.py

```diff
@@ -59,11 +59,17 @@ def __init__(self):
 @click.option(
     '--cephfs', is_flag=True,
     help='Set cephfs volume for cluster storage.')
+@click.option(
+    '--cephfs-size',
+    type=int,
+    default=200,
+    help='Set cephfs volume size in GB.')
 @click.option(
     '--debug', is_flag=True,
     help='If set, deploy REANA in debug mode.')
 @click.pass_context
-def cli(ctx, loglevel, skip_validation, file, cvmfs, cephfs, debug):
+def cli(ctx, loglevel, skip_validation, file, cvmfs,
+        cephfs, cephfs_size, debug):
```

Comment:
Shall we put in place some warning in case user sets `--cephfs-size` but no `--cephfs`? Or if `--cephfs-size` is set, silently set `--cephfs` to true?